prac_08/KivyDemos-master/dynamic_labels.py

```diff
@@ -0,0 +1,28 @@
+"""
+CP1404 - Practical 8 - 5. Dynamic Labels
+"""
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.label import Label
+
+
+class DynamicLabelsApp(App):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.names = ["Ewa", "Mari", "Moni", "Sarah", "Sophie", "Emma", "Megan"]
+
+    def build(self):
+        """Build the Kivy GUI."""
+        self.title = "Dynamic Labels"
+        self.root = Builder.load_file('dynamic_labels.kv')
+        self.create_labels()
+        return self.root
+
+    def create_labels(self):
+        """Create buttons from data and add them to the GUI."""
```

Comment:
This doc-string is inaccurate as the function creates labels

Reply:
Thank you for noticing! I have changed it!